src/js/components/organisms/hero-cover/hero-cover.scss

```diff
@@ -0,0 +1,43 @@
+/** @format */
+
+.hero--cover {
+    @extend %reset-last-margin;
```

Comment:
This needs to go.

Reply:
Not going into css yet. Only moving components for now. Css is the next stage / stage.